docs/central-users.rst

```diff
@@ -80,12 +80,6 @@ By default, Central roles are configured to allow the following:
 +------------------+---------------+-----------------+----------------+----------------+
 | Delete           | x             |                 |                |                |
 +------------------+---------------+-----------------+----------------+----------------+
-| **Server Configuration**                                                             |
```

Comment:
I removed the entire "Server Configuration" section, but there is one server configuration action that is still available: setting up usage information reporting. I've filed the related issue #1561.